src/util/pointer_offset_size.cpp

```diff
@@ -38,14 +39,14 @@ optionalt<mp_integer> member_offset(
     {
       const std::size_t w = to_c_bit_field_type(comp.type()).get_width();
       bit_field_bits += w;
-      result += bit_field_bits / 8;
-      bit_field_bits %= 8;
+      result += bit_field_bits / config.ansi_c.char_width;
```

Comment:
I'm assuming we guarantee that char_width is always different than zero in all these cases, right?

Reply:
Yes, though we might one day want to have some architecture sanity check.

Reply:
It can be loaded from file through `configt::set_from_symbol_table` and `symtab2gb`. Therefore it could theoretically be set to 0 in the file and I am not sure we have appropriate validation that the value from file is not 0.